src/View/Helper/HtmlHelper.php

```diff
@@ -5,6 +5,38 @@ class HtmlHelper extends \Cake\View\Helper\HtmlHelper
 {
     use OptionsAwareTrait;
 
+    /**
+     * A list of allowed styles for badges.
+     *
+     * @var array
+     */
+    public $badgeClasses = [
+        'primary', 'badge-primary',
+        'secondary', 'badge-secondary',
+        'success', 'badge-success',
+        'danger', 'badge-danger',
+        'warning', 'badge-warning',
+        'info', 'badge-info',
+        'light', 'badge-light',
+        'dark', 'badge-dark',
+    ];
+
+    /**
+     * A mapping of aliases for badges.
+     *
+     * @var array
+     */
+    public $badgeClassAliases = [
```

Comment:
Similar comment as above, this array is a duplicate of `OptionsAware::$buttonClassAliases`.